std_msgs/Empty empty
---
sensor_msgs/PointCloud2 camera_cloud
